dash_core_components/__init__.py

```diff
@@ -22,6 +22,11 @@
 _this_module = _sys.modules[__name__]
 
 _js_dist = [
+	{
+        'external_url': 'https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.4/MathJax.js?config=TeX-MML-AM_CHTML',
+        'relative_package_path': 'mathjax-2.7.4.js',
```

Comment:
I believe this should be `mathjax-2.7.4.min.js`

Reply:
Or vice-versa, everything could be labeled `mathjax-2.7.4.js` rather than `mathjax-2.7.4.min.js`. I don't really have a preference, but they do need to all have the same name 😄

Reply:
Uh my bad. I'll change it in MANIFEST.in